bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java

```diff
@@ -216,8 +200,14 @@ public void start() {
 	@Override
 	public void hearShake() {
 		logger.d("Shake detected!");
-		feedbackEmailFlowManager.startFlowIfNeeded(application,
-			false);
+		String str = application.getClass().toString();
+		if (str.equals("class com.expedia.bookings.activity.ExpediaBookingApp")) {
```

Comment:
Originally we had this code in place because we wanted to prevent the app from taking another screenshot if you were already on the annotation activity, but we couldn't figure out how to make it work anyway so we just deleted this part